src/main/java/org/apache/maven/buildcache/CacheControllerImpl.java

```diff
@@ -112,13 +114,10 @@
 @SuppressWarnings("unused")
 public class CacheControllerImpl implements CacheController {
 
-    public static final String FILE_SEPARATOR_SUBST = "_";
-    /**
-     * Prefix for generated sources stored as a separate artifact in cache
-     */
-    private static final String BUILD_PREFIX = "build" + FILE_SEPARATOR_SUBST;
-
     private static final Logger LOGGER = LoggerFactory.getLogger(CacheControllerImpl.class);
+    private static final String DEFAULT_FILE_GLOB = "*";
+    public static final String ERROR_MSG_RESTORATION_OUTSIDE_PROJECT =
+            "Blocked an attempt to restore files outside of a project directory : ";
```

Comment:
in english, no space before `:` (yes, french with non-breaking space is much better, but we must adapt :) )

Reply:
Gosh, I should know, it is precisely the reason how I got a Apache Jira account (and my very first contribution to the open source world 😋) : https://issues.apache.org/jira/browse/CXF-7185